swift-sdk/Internal/AuthManager.swift

```diff
@@ -53,6 +53,12 @@ class AuthManager: IterableAuthManagerProtocol {
         }
     }
     
+    func setNewTokenAfterUpdateEmail(_ newToken: String) {
```

Comment:
I think we should change the method name to reflect more what it actually does. It appears that this method does not need to be aware that it is called after update email.